src/CryptoAxus.API/Controllers/RootController.cs

```diff
@@ -33,7 +33,11 @@ public async Task<ActionResult<BaseResponse<IReadOnlyList<Links>>>> GetRoot()
 
                 new Links(href: Url.RouteUrl("PostArtist"),
                           rel: "post",
-                          method: Constants.PostMethod)
+                          method: Constants.PostMethod),
+
+               new Links(href: Url.RouteUrl(routeName: "GetArtistByWalletAddress", values: new { userWalletAddress = "0x647115d2b38bc8ea242beb01" }),
+                          rel: "artist",
```

Comment:
Rel field should be get_userWalletAddress

Reply:
Resolved